src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -40,6 +40,14 @@
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Builder.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Enumerator.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.CollectionInterfaces.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.IImmutableList.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.IndexOf.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.Insert.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.OfType.cs" />
```

Comment:
It does seem a bit overkill to separate all of these methods in their own files like `Insert`, `Remove`, and `Sort`. Why not just have two files: one file with the stuff you want so you can build it without higher APIs, and one file with everything else?